demo/scripts/installCC.sh

```diff
@@ -10,11 +10,38 @@ set -ev
 
 export WAIT_TIME=15
 export version=1.0
-export FABRIC_BIN_DIR=/project/src/github.com/hyperledger/fabric/build/bin
-export PEER_CMD=/project/src/github.com/hyperledger-labs/fabric-private-chaincode/fabric/bin/peer.sh
+export FPC_PATH=/project/src/github.com/hyperledger-labs/fabric-private-chaincode
+export FABRIC_BIN_DIR=${FPC_PATH}/../../hyperledger/fabric/build/bin
+export PEER_CMD=${FPC_PATH}/fabric/bin/peer.sh
 
-docker exec peer0.org1.example.com ${PEER_CMD} chaincode install -n auctioncc -v ${version} --path demo/chaincode/fpc/_build/lib -l fpc-c
-sleep ${WAIT_TIME}
+# Command to execute peer cli inside peer container 
+# Note: by default the peer runs with peer credentials, so we have to override CORE_PEER_MSPCONFIGPATH env-var that the peer in cli-mode doesn't use peer but admin credentials
+REMOTE_PEER_CMD="docker exec -e CORE_PEER_LOCALMSPID=Org1MSP -e CORE_PEER_MSPCONFIGPATH=/etc/hyperledger/msp/users/Admin@org1.example.com/msp peer0.org1.example.com env TERM=${TERM} ${PEER_CMD}"
+REMOTE_CAT="docker exec peer0.org1.example.com cat"
 
-docker exec peer0.org1.example.com ${PEER_CMD} chaincode instantiate -n auctioncc -v ${version} --channelID mychannel -c '{"Args":[]}'
-sleep ${WAIT_TIME}
+# Commands to execute cli _outside_ peer container
```

Comment:
Eventually we should have cli running from client host rather than docker execing into peer.  Doesn't 100% work yet, so no default but unless you have strong objection i would prefer to keep these comments there ...

Reply:
sounds good